helm_deploy/values-dev.yaml

```diff
@@ -33,9 +33,9 @@ env:
 
   SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: "https://gateway.t3.nomis-api.hmpps.dsd.io/auth/.well-known/jwks.json"
 
-  OFFENDER_DELETION_CRON: "0 0 10 21 4 ? *"
+  OFFENDER_DELETION_CRON: "0 0 9 22 4 ? *"
   OFFENDER_DELETION_INITIAL_WINDOW_START: "2000-05-10T00:00:00"
-  OFFENDER_DELETION_WINDOW_LENGTH: "P1D"
+  OFFENDER_DELETION_WINDOW_LENGTH: "P2427D"
```

Comment:
This is a window from 2000-05-10 to 2007-01-01 that will cause 63 offenders in T3 to be deleted